docs/Using Randomness/example-contracts.md

```diff
@@ -56,26 +55,30 @@ To use this contract, compile and deploy it in Remix.
 
-1. Fund the subscription by running the `topUpSubscription()` function. For this function, a value of 1000000000000000000 = 1 LINK.
+1. In this example, the `topUpSubscription()` function sends LINK from your contract to the subscription. Fund your contract with at least three testnet LINK. Alternatively, you can send LINK directly to the subscription in the [Subscription Manager](https://vrf.chain.link). Any address can provide funding to a subscription balance. If you need testnet LINK, you can get it from one of the available [Rinkeby faucets](/docs/link-token-contracts/#rinkeby).
 
-1. Create and deploy consumer contracts that include the following components:
+1. Run the `topUpSubscription()` function to send LINK from your contract to your subscription balance. For this example, specify a value of `3000000000000000000`, which is equivalent to three LINK.
 
-    - The `requestRandomWords()` function and the required variables
+1. Create and deploy a consumer contract that includes the following components:
+
+    - The `requestRandomWords()` function and the required variables and your subscription ID
     - The `fulfillRandomWords()` callback function
 
-1. After you deploy these consumer contracts, add them to the subscription as approved consumers using the `addConsumer()` function on your subscription manager contract.
+    You can use the example from the [Get a Random Number](/docs/get-a-random-number/#analyzing-the-contract) guide.
+
+1. After you deploy the consumer contract, add it to the subscription as an approved consumer using the `addConsumer()` function on your subscription manager contract. Specify the address for your consumer contract.
 
-1. On each of the consumer contracts, run their `requestRandomWords()` functions to request and receive random values.
+1. On the consumer contract, run the `requestRandomWords()` functions to request and receive random values. The request might take several minutes to process. Track pending request status in the [Subscription Manager](https://vrf.chain.link) app.
 
-1. When you are done with this contract and its subscription, run the `cancelSubscription()` to close the subscription and send remaining LINK tokens to your wallet address.
+    The consumer contract can continue to make requests until your subscription balance runs out. The subscription manager contract must maintain sufficient balance in the subscription so that the consumers can continue to operate.
 
-The consumer contracts can continue to make requests until your subscription balance runs out. The subscription manager contract must maintain sufficient balance in the subscription so that the consumers can continue to operate.
+1. When you are done with your contracts and the subscription, run the `cancelSubscription()` to close the subscription and send the remaining LINK to your wallet address.
```

Comment:
maybe add a sentence to emphasize that the function requires a parameter:

Specify the address of the receiving wallet.